src/components/ForgotPassword.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom'
+import styles from '../../dist/panel.less';
+
+import SignUp from './SignUp';
+import LogIn from './LogIn';
+
+const ForgotPassword = () => (
+	<div className="center-container">
```

Comment:
So forgotpassword, signup, and login pages have same format with different contents.
We could just reuse a global pure component and pass in contents as props